src/relay/backend/contrib/dnnl/codegen.cc

```diff
@@ -507,6 +540,9 @@ class DNNLJSONSerializer : public backend::contrib::JSONSerializer {
         std::vector<std::string> op_list = ParsingOpList(name);
         call = GetRootCall(fn->body.as<CallNode>(), op_list.size() - 1, op_list);
         ICHECK(call->op.as<OpNode>()) << "Not op node";
+      } else if (name.find("gelu") != std::string::npos) {
```

Comment:
Minor objection. GELU may be fused not only with dense. I see that you care only about BERT like topologies and there is no other real topologies using that activation. But in general that is not fully correct.

Reply:
Hi @apeskov , I agree! For generalization purpose, we should extend gelu fusion to other operators such as conv. But for this patch, I would like to make it dedicated to enabling dense+bias+gelu. I will create a issue to track this and try to propose another patch to cover the case. The new pattern following with `gelu` is only dispatched by dense operator so I made an assumption here.

Reply:
I see. In context of this patch it's functionally correct. Just counterintuitive `if` statement. Check if contain "gelu" and after that search "dense". Without additional explanation it may make some engineers a slightly confused.

So some additional comment line here would be great. Anyway that's is not blocker.